Assignment _5/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.3)
+project(Assignment__5)
+
```

Comment:
Don't forget to include your lib folder if this is you "driver"
`include_directories( ./lib )
link_directories( ./lib )`